app/src/main/res/layout/activity_main.xml

```diff
@@ -79,15 +79,26 @@
         android:text="@string/humidity"
         android:textColor="@color/black"
         app:layout_constraintStart_toStartOf="@id/high"
-        app:layout_constraintTop_toBottomOf="@id/high"/>
+        app:layout_constraintTop_toBottomOf="@id/high" />
 
     <TextView
         android:id="@+id/pressure"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="@string/pressure"
         android:textColor="@color/black"
-        app:layout_constraintTop_toBottomOf="@id/humidity"
-        app:layout_constraintStart_toStartOf="@id/humidity"/>
+        app:layout_constraintStart_toStartOf="@id/humidity"
+        app:layout_constraintTop_toBottomOf="@id/humidity" />
+
+    <Button
+        android:id="@+id/forecast_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:onClick="forecastActivity"
```

Comment:
This works, but it's bad practice because it mixes declarative UI (i.e. XML) with view logic (i.e. `setOnClickListener`). In the future (until we get to view binding), use `findViewById` coupled with `view.setOnClickListener {}`.